x-pack/test/accessibility/apps/group1/dashboard_panel_options.ts

```diff
@@ -18,7 +17,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   // Failing: See https://github.com/elastic/kibana/issues/147667
   describe.skip('Dashboard panel options a11y tests', () => {
```

Comment:
Outside the scope of this PR, but I see this whole test suite is skipped for failing tests. 🙈  But good idea to update the methods anyways. 💯